definitions/ext-service/definition.yml

```diff
@@ -8,18 +8,59 @@ synthesis:
       encodeIdentifierInGUID: true
       conditions:
         - attribute: service.name
+
     # telemetry with service_name attribute
     - identifier: service_name
       name: service_name
       encodeIdentifierInGUID: true
       conditions:
         - attribute: service_name
+
     # telemetry with serviceName attribute
     - identifier: serviceName
       name: serviceName
       encodeIdentifierInGUID: true
       conditions:
         - attribute: serviceName
+
+    # telemetry from opentelemetry provider
+    - identifier: service.name
```

Comment:
This is already being matched in the rule defined in L6.  I'm actually wondering if we should remove that one, and keep this one which is more specific.  From what I know the original intent of supporting `service.name` was OTEL so it allows us to match with more precission.  We were already seeing that a raw `service.name` match was too broad.